pkg/volume/empty_dir/empty_dir_test.go

```diff
@@ -42,7 +42,12 @@ func makePluginUnderTest(t *testing.T, plugName, basePath string) volume.VolumeP
 }
 
 func TestCanSupport(t *testing.T) {
-	plug := makePluginUnderTest(t, "kubernetes.io/empty-dir", "/tmp/fake")
+	tmpDir, err := ioutil.TempDir("/tmp", "fake")
```

Comment:
I recommend using [```os.TempDir()```](https://golang.org/pkg/os/#TempDir) instead of the string ```/tmp``` in the case ```/tmp``` isn't the local temporary directory.